package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
   <name>rqt_common_plugins</name>
-  <version>0.4.8</version>
+  <version>1.0.0</version>
```

Comment:
Please don't update the version yet. That happens when `catkin_prepare_release` is being run.